config/default/mutatepods_opt_in_patch.yaml

```diff
@@ -0,0 +1,23 @@
+# mutatepods opt-in mode
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+ name: mutator
+webhooks:
+  - name: mutatepods.kubemacpool.io
+    namespaceSelector:
+      matchExpressions:
+        - key: runlevel
+          operator: NotIn
+          values:
+            - "0"
+            - "1"
+        - key: openshift.io/run-level
+          operator: NotIn
+          values:
+            - "0"
+            - "1"
```

Comment:
These two must be always on. We can use JsonPatch to only set the mutateX.kubemacpool.io one

Reply:
thing is, if I use `patchesStrategicMerge` (like I currently do) then I have to set the  `runlevel` keys otherwise they are removed.
But, if I use `JasonPatch` then I will need to split `mutatepods_opt_out_patch` to 2 patches because it handles 2 instances - the mutatingwebhookconfiguration and the KMP Namespace which needs to be opted out to avoid deadlock)

Between the two options I chose patchesStrategicMerge because it keeps everything in the same patch for better understanding. But if you think it's better then I'll change it to JasonPatches.

Reply:
All good then. We documented that it has to be patched, so i think we are good.